src/components/form/Form/Form.jsx

```diff
@@ -87,13 +87,10 @@ export default function Form({
     scrollToError,
     ...formProps
 }) {
+    const formRef = useRef();
     useEffect(() => {
         if (scrollToError && !_.isEmpty(errors)) {
-            // eslint-disable-next-line react/no-find-dom-node
-            const formElement = ReactDOM.findDOMNode(this);
-            if (formElement) {
-                formElement.scrollIntoView({ behavior: 'smooth', block: 'start' });
-            }
+            formRef.current?.scrollIntoView?.({ behavior: 'smooth', block: 'start' });
```

Comment:
The optional chaining that checks whether `scrollIntoView` is available before calling it is for unit tests - it does not exist in jsdom and that results in some tests failing.

Alternatively, I can disable the `scrollToError` feature in the tests and leave this as

```suggestion
            formRef.current?.scrollIntoView({ behavior: 'smooth', block: 'start' });
```